device/sycl/src/seeding/seed_finding.sycl

```diff
@@ -172,17 +177,59 @@ vecmem::data::vector_buffer<seed> seed_finding::operator()(
         doublet_counts.begin(), doublet_counts.end(), mb_buffer_sizes.begin(),
         [](const device::doublet_counter_header& dc) { return dc.m_nMidBot; });
 
-    // create the triplet_counter container with the number of doublets
+    // Get the prefix sum for the midBot doublets using buffer.
+    const device::prefix_sum_t mb_prefix_sum = device::get_prefix_sum(
+        mb_buffer_sizes, (m_mr.host ? *(m_mr.host) : m_mr.main));
```

Comment:
Instead of introducing a whole new overload for `traccc::device::get_prefix_sum`, let's use the existing function like:

```c++
const device::prefix_sum_t mb_prefix_sum = device::get_prefix_sum(
    std::vector<unsigned int>(mb_buffer_sizes.begin(), mb_buffer_sizes.end()),
    (m_mr.host ? *(m_mr.host) : m_mr.main));
```

Or something like that... I didn't try exactly what the formalism should be. The main point is that you can convert between different integer type vectors with a single expression.